lib/openstudio-standards/schedules/parametric.rb

```diff
@@ -555,6 +564,8 @@ def self.schedule_ruleset_get_parametric_inputs(schedule_ruleset, space_load_ins
           # skip if rules already match
           if (sch_ruleset_days_used[sch_index] - day_group).empty?
             OpenStudio.logFree(OpenStudio::Debug, 'openstudio.standards.Parametric.Schedules', "in #{__method__}: #{schedule_ruleset.name} rule #{sch_index} already matches hours of operation rule #{hoo_index}; new rule won't be created.")
+            # iterate new_rule_ct anyway to keep these rules
+            new_rule_ct += 1 unless sch_index == -1
```

Comment:
Iterating this counter when rules already match an hours_of_operation rule (except when it's the default profile) means those rules won't inadvertently be removed below.